consensus/istanbul/core/prepare.go

```diff
@@ -98,7 +111,8 @@ func (c *core) verifyPreparedCertificate(preparedCertificate istanbul.PreparedCe
 		// If COMMIT message, verify valid committed seal.
 		if message.Code == istanbul.MsgCommit {
 			_, src := c.valSet.GetByAddress(signer)
-			err := c.verifyCommittedSeal(subject.Digest, message.CommittedSeal, src)
+
```

Comment:
Would it be cleaner to move this block into the `if message.Code == istanbul.MsgCommit` above?